lib/beaker/local_connection.rb

```diff
@@ -53,9 +53,12 @@ def execute command, options = {}, stdout_callback = nil, stderr_callback = stdo
           result.stdout << std_out
           result.stderr << std_err
           result.exit_code = status.exitstatus
+          @logger.info(result.stdout)
+          @logger.info(result.stderr)
```

Comment:
How do you know what's what? Is that important?

Also, what happens if either is empty? Will it log empty lines?

Reply:
It's important to print them in the right order, as a command might right stuff on stdout and stderr.
And yes, if either is empty, an empty line is printed.
I'll add a check and print to log, only if there is something to print.

Reply:
@ekohl , @ghoneycutt are there any more issues with this PR?